app/controllers/api/likes_controller.rb

```diff
@@ -0,0 +1,20 @@
+class Api::LikesController < ApplicationController
+  def create
+    if already_liked?
+      render json: { message: "You can't like more than once" }, status: 422
```

Comment:
we are not supposed to use this message in the front end right...??

Reply:
It is not a necessity to display it in the frontend!